pkg/serverless/invocationlifecycle/lifecycle_test.go

```diff
@@ -57,8 +57,9 @@ func TestStartExecutionSpanNoLambdaLibrary(t *testing.T) {
 	mockProcessTrace := func(*api.Payload) {}
 	mockDetectLambdaLibrary := func() bool { return false }
 
+	EventPayload := `a5a{"resource":"/users/create","path":"/users/create","httpMethod":"GET","headers":{"Accept":"*/*","Accept-Encoding":"gzip","x-datadog-parent-id":"1480558859903409531","x-datadog-sampling-priority":"1","x-datadog-trace-id":"5736943178450432258"}}0`
```

Comment:
When you capitalize a variable in Go, it becomes available from outside of that `package` (group of files). Lowercased variables are only available within the same `package`.

In this case `EventPayload` is capitalized, but it won't be available outside of the package because it's inside a function. Nonetheless, I would lowercase the first letter just to avoid confusion, because capitalization has a special meaning in Go.